test/moonwall.config.json

```diff
@@ -388,9 +388,52 @@
         ]
       }
     },
+    {
+      "name": "dev_moonriver",
+      "testFileDir": ["suites/dev/common"],
```

Comment:
we may have to add "/suites/dev/moonriver" in the future if we have specific tests for it